python/pyxir/contrib/target/components/common/op_support.py

```diff
@@ -167,7 +167,7 @@ def is_conv2d_transpose_supported(
 
 def is_padding_supported(X: XLayer, bXs: List[XLayer], tXs: List[XLayer]) -> bool:
 
-    if len(tXs) == 1 and tXs[0].type[0] in ["Pooling", "Convolution"]:
+    if X.attrs["pad_value"] == 0.0 and len(tXs) == 1 and tXs[0].type[0] in ["Pooling", "Convolution"]:
```

Comment:
Following DPUs are not using this function:
- https://github.com/Xilinx/pyxir/blob/57c402361aeadc6360344341e83af134ea21b5de/python/pyxir/contrib/target/components/DPUCADF8H/dpu_op_support.py#L195
- https://github.com/Xilinx/pyxir/blob/57c402361aeadc6360344341e83af134ea21b5de/python/pyxir/contrib/target/components/DPUCAHX8H/u280_op_support.py#L203
- https://github.com/Xilinx/pyxir/blob/57c402361aeadc6360344341e83af134ea21b5de/python/pyxir/contrib/target/components/DPUCAHX8H/u50_op_support.py#L202
- https://github.com/Xilinx/pyxir/blob/57c402361aeadc6360344341e83af134ea21b5de/python/pyxir/contrib/target/components/DPUCZDX8G/som_op_support.py#L202
- https://github.com/Xilinx/pyxir/blob/57c402361aeadc6360344341e83af134ea21b5de/python/pyxir/contrib/target/components/DPUCZDX8G/ultra96_op_support.py#L201
- https://github.com/Xilinx/pyxir/blob/57c402361aeadc6360344341e83af134ea21b5de/python/pyxir/contrib/target/components/DPUCZDX8G/zcu102_op_support.py#L201

Could you also add support there?